src/main/default/lwc/subjectInput/subjectInput.html

```diff
@@ -0,0 +1,29 @@
+<template>
+  <label for="subject-input">
+    <c-section-title title={label}></c-section-title>
+  </label>
+  <div class="slds-is-relative">
+    <div class={formClass}>
```

Comment:
Is it necessary to have this div and it's direct child div or could they be merged into one with `formClass` including "slds-form-element__control"?

Reply:
I followed the same structure from the SLDS blueprint but looking at what the CSS classes contains, in my case I can get rid of one of the divs so I will update this 👌